src/include/rbd/librbd.hpp

```diff
@@ -75,15 +121,22 @@ namespace librbd {
 
   typedef struct {
     std::string name;
-    int64_t pool;
+    std::string pool;
     group_image_state_t state;
   } group_image_status_t;
 
   typedef struct {
     std::string name;
-    int64_t pool;
+    std::string pool;
```

Comment:
Yup -- but I skipped over it since I knew you would have to refactor it to eliminate the variant.